jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -194,6 +205,28 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
            name: 'tenants',
          },
        ] else []) +
+      (if api.config.tenants.tenants != {} then [
+          {
+           secret: {
+             secretName: tenant.mTLS.secretName,
+           },
+           name: tenant.name + '-mtls-secret',
+          },
+          for tenant in api.config.tenants.tenants
+          if std.objectHas(tenant, 'mTLS')
+          if std.objectHas(tenant.mTLS, 'secretName')
+       ] else []) +
+      (if api.config.tenants.tenants != {} then [
+          {
+           configMap: {
+             name: tenant.mTLS.configMapName,
+           },
+           name: tenant.name + '-mtls-configmap',
+          },
+          for tenant in api.config.tenants.tenants
+          if std.objectHas(tenant, 'mTLS')
+          if std.objectHas(tenant.mTLS, 'configMapName')
```

Comment:
I think it would be clearer if we avoided this duplication and changed this to and `else if` statement above.